src/Helper/MenuHelper.php

```diff
@@ -45,6 +45,16 @@ protected function pruneAccessDeniedEntries(array $menuConfig, array $entitiesCo
                 unset($menuConfig[$key]);
                 continue;
             }
+            else if (
+                'route' === $entry['type']
+                && isset($entry['role'])
+                && !$this->adminAuthorizationChecker->isEasyAdminGranted(
```

Comment:
Please use Symfony `security.authorization_checker` service. `AdminAuthorizationChecker` is intended to check access regarding an entity EasyAdmin config. You can not invent a list action role to deal with this case.

Reply:
Adding a test to cover the use case would be really kindful too ;-)

Reply:
Necessary? Now it's very simple and uses authorizationChecker:
```php
if (
    'route' === $entry['type']
    && isset($entry['role'])
    && !$this->authorizationChecker->isGranted($entry['role'])
)
```

Reply:
Necessary? Now it's very simple and uses authorizationChecker:
```php
if (
    'route' === $entry['type']
    && isset($entry['role'])
    && !$this->authorizationChecker->isGranted($entry['role'])
)
```